test/test_ResponseHandler.py

```diff
@@ -333,9 +349,54 @@ def test_set_response(self, db, utils, max_commands, monkeypatch):
                             expected_status = 2
                         assert part[key]["status"] == expected_status
                         if key == "AddImage":
-                            assert(self.response_blobs[set_key][j]
-                                   == j * 2)
+                            assert (self.response_blobs[set_key][j]
+                                    == j * 2)
                             expected_blobs = expected_blobs + 1
                         else:
                             assert part[key]["returned"] == j + 1
                 assert len(self.response_blobs[set_key]) == expected_blobs
+
+    @pytest.mark.parametrize("cpq", range(1, 3))
+    def test_varying_response_w_index(self, db, utils, cpq):
+        self.cleanDB()
+        persons = EntityWithResponseDataCSV(
+            "./input/persons.adb.csv", self.requests, self.responses)
+        loader = ParallelLoader(db)
+        loader.ingest(persons, batchsize=99,
+                      numthreads=31,
+                      stats=True)
+        assert loader.error_counter == 0
+
+        dist = persons.df.groupby(persons.df.age // 10 * 10).count()
```

Comment:
Is this variable ever used?

Reply:
no, removed. I changed how the code calculated.